packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -82,6 +82,36 @@ export const bm25s = (model: ModelData): string[] => [
 retriever = BM25HF.load_from_hub("${model.id}")`,
 ];
 
+export const depth_anything_v2 = (): string[] => [
+	`git clone https://github.com/DepthAnything/Depth-Anything-V2
+
+cd Depth-Anything-V2
+pip install -r requirements.txt
```

Comment:
Maybe return the shell commands as one snippet and the Python code as another one (see how the transformers pipeline / automodel snippets are rendered, for instance)